tests/test_metadata.py

```diff
@@ -782,3 +784,50 @@ def filter_bbox(self, bbox):
     assert isinstance(new, MyCubeMetadata)
     assert orig.bbox is None
     assert new.bbox == (1, 2, 3, 4)
+
+
+collection_json = {
```

Comment:
I would not define these inputs as top level variables, as we are probably going to add a lot of variations

Reply:
Ok, so I'll just plug the dictionaries directly into the @pytest.mark.parametrize decorator?

Reply:
indeed, I would start with that,

In a further phase you could also define a helper function to dynamically generate stac-like dicts for various (parameterized) tests